### PR TITLE
P2022 703

### PR DESCRIPTION
**Pull request by PienkowskaMarta**
Adds navigation from login form to home screen.

### CODE REVIEW

**Review by Lucas-Fresko — COMMENTED** [review]
Here's a link explaining the .idea folder: https://betterprogramming.pub/exploring-the-idea-folder-790931c19191

The deploymentTargetDropDown.xml stores your emulator target device, which I'm guessing we don't need in the remote repository.

The codeStyles are apparently a good idea to have, but if we decide to implement it on remote, it should probably have its own task and PR.

**Review by taurelas — CHANGES_REQUESTED** [review]
please reset the branch to master and commit only final changes from the LoginActivity as advised on Slack

**Review by Lucas-Fresko — APPROVED** [review]
Nicely done!

File: app/src/main/java/com/intive/patronage22/intivi/LoginActivity.kt [hunk 1]
```diff
@@ -7,5 +7,15 @@ class LoginActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
+        findViewById<Button>(R.id.loginButton).setOnClickListener(listener)
```
Comment (taurelas) [inline app/src/main/java/com/intive/patronage22/intivi/LoginActivity.kt:10]: you forgot to copy the imports! :) they are usually collapsed in Android Studio so they're easy to miss.
For the future - It's always a good idea to run the app before the commit and ensure it compiles
Please do so and fix the issues

**Review by taurelas — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/intive/patronage22/intivi/LoginActivity.kt (hunk 1 above)
Comment (PienkowskaMarta) [reply app/src/main/java/com/intive/patronage22/intivi/LoginActivity.kt:15]: Thanks for noticing that! I have added missing imports. It works now.

**Comment by AdrianPat** [comment]
There is a shorter way:

```
findViewById<Button>(R.id.loginButton).setOnClickListener {
    startActivity(Intent(applicationContext, MainActivity::class.java))
}
```

It works. What about this code?

**Comment by Lucas-Fresko** [comment]
Is the reusability intended to come from implementing cases for different id's in the 'when ->' ?

I played around with;
/* private fun goToActivity(activityToGoTo: Class<*>?){
        startActivity(Intent(this, activityToGoTo))
    }
*/

and;
` findViewById<Button>(R.id.loginButton).setOnClickListener{goToActivity(MainActivity::class.java)} `

But I'm not sure if there will ever be a need to go to MainActivity from more than one place so I'd be satisfied with just one line, while your listener can be used more flexibly by calling other functions from it.

**Review by taurelas — APPROVED** [review]

**Pull request merged**